src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -16,41 +16,33 @@ This page describes how resource usage, pricing, and cluster configurations work
+## Provisioned capacity
 
-RU and storage consumption is prorated at the following prices:
+Provisioned capacity refers to the processing resources (Request Units per sec) reserved for your workload. Each 500 RUs/sec equals approximately 1 vCPU. 
 
-  Unit                    | Cost
-  ------------------------|------
-  1M Request Units        | $0.20
-  1 GiB storage           | $0.50
+Estimate your workload's peak vCPU needs by analyzing available historical data, adjusted for future changes, or by comparing with similar existing workloads. We recommend setting capacity at least 40% above expected peak workload to avoid performance issues. 
 
-Refer to [Pricing](https://cockroachlabs.com/pricing) to see cost estimates of common queries and how they increase with the size and complexity of the query. You can view your cluster's RU and storage usage on the [**Cluster Overview** page]({% link cockroachcloud/cluster-overview-page.md %}).
+You can scale the provisioned capacity up or down based on workload changes, allowing for efficient resource management and cost optimization.
 
-## Free vs. paid usage
+For [multi-region deployments](#multi-region-clusters), the single provisioned capacity value you configure for the cluster applies across all regions, acting as an overall capacity budget from which each region can draw depending on its processing requirements.
 
-CockroachDB {{ site.data.products.standard }} clusters scale based on your workload so that you will only pay for what you use beyond the free resources. Each non-contract CockroachDB {{ site.data.products.cloud }} organization is given 50 million [Request Units](#request-units) and 10 GiB of storage for free each month. Free resources do not apply to contract customers. Free resources can be spent across all CockroachDB {{ site.data.products.standard }} clusters in an organization and will appear as a deduction on your monthly invoice.
+## Pricing
 
-Setting resource limits will allow your cluster to scale to meet your application's needs and maintain a high level of performance. You must [set resource limits]({% link cockroachcloud/cluster-management.md %}#edit-cluster-capacity) if you've already created one free CockroachDB {{ site.data.products.standard }} cluster. To set your limits, you can either set storage and RU limits individually, or enter a dollar amount that will be split automatically between both resources. You can also choose an unlimited amount of resources to prevent your cluster from ever being throttled or disabled.
-  
-## Choose resource limits
+CockroachDB {{ site.data.products.standard }} pricing is determined by two components: provisioned capacity and storage.
```

Comment:
nit: I would maybe lay this out with "provisioned capacity and storage" as bullet points under this first sentence, with each of the bullet points linking to the section underneath. It may not be totally necessary right now, but could help to future proof it as the sections may become larger.

Reply:
That's how I had written it first, but it felt weird to have only two bullet points. I think I'll leave this as is for now. We can revisit it if we add more pricing elements (I don't foresee any additional ones being added anytime soon)